getting_started.Rmd

```diff
@@ -97,9 +97,9 @@ If you have organization admin privileges and plan on making multiple courses wi
 
 - Click `New repository secret`/`New organization secret` button.
 - Under `Name` you must use `GH_PAT`.
-- For `Secret`: Create a personal access token [following these instructions](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token). 
+- For `Secret`: Create a personal access token [following these instructions](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token).
 - Underneath `Select scopes`, check both `repo` and `workflow`.
-- Then copy the PAT and save as the secret. 
+- Then copy the PAT and save as the secret.
```

Comment:
```suggestion
- Then copy the PAT and save as a secret in the repository that you made from our template.
```

Reply:
At this point it isn't called GH_PAT, but I agree calling it PAT may be confusing. We should probably just spell out that it is a personal access token.